datatable/readXlsxFromSharepointConfig.tsx

```diff
@@ -0,0 +1,88 @@
+import { tl } from "processhub-sdk/lib/tl.js";
+import { Language } from "processhub-sdk/lib/tl.js";
+import { DataTableErrorCode } from "./common.js";
+import { SharePointConfigFields, SharePointConfigFieldsHelp } from "../sharepoint/readfile-config.js";
+
+export function readXlsxFromSharepointConfig(userLanguage: Language): React.JSX.Element {
+  return (
+    <div id="service-form" className="ui form center">
+      <table className="table table-striped table-bordered">
+        <tbody>
+          <SharePointConfigFields userLanguage={userLanguage} />
+
+          <tr>
+            <td>
+              <span>{tl("Name des Arbeitsblatts (optional)", userLanguage)}</span>
+            </td>
+            <td>
+              <input id="sheetName" />
+            </td>
+          </tr>
+
+          <tr>
+            <td>
+              <span>{tl("Filter (optional)", userLanguage)}</span>
+            </td>
+            <td>
+              <input id="rowFilter" />
+            </td>
+          </tr>
+
+          <tr>
+            <td>
+              <span>{tl("Ergebnisfeld", userLanguage)}</span>
+            </td>
+            <td>
+              <select id="dataTableField" />
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+      <table className="table">
+        <tbody>
+          <tr>
+            <td colSpan={2}>
+              <SharePointConfigFieldsHelp userLanguage={userLanguage} />
+              <h3>{tl("Name des Arbeitsblatts", userLanguage)}</h3>
+              <div>
+                <p>{tl("Optional: Der Name des Arbeitsblatts, das ausgelesen werden soll. Ist der Name leer, wird immer das erste Arbeitsblatt ausgelesen.", userLanguage)}</p>
+              </div>
+              <h3>{tl("Filter", userLanguage)}</h3>
+              <div>
+                <p>
+                  {tl(
+                    "Optional: Ein JavaScript-Ausdruck zum Filtern der Eingabedatei. Dieser Filter wird auf jede Zeile angewendet. Dabei kann über das Ojekt field auf die Felder des Vorgangs und über das Objekt row auf die Spalten der aktuellen Zeile zugegriffen werden.",
```

Comment:
Typo: Ojekt  -> Objekt .
Evlt. field und row in Anführungszeichen, damit man erkennt das hier Bezeichner gemeint sind.